.codecov.yml

```diff
@@ -0,0 +1,29 @@
+# -- repository yaml --
+
+# Explicitly wait for all jobs to finish, as wait_for_ci prematurely triggers.
+# See https://github.com/python-trio/trio/issues/2689
+codecov:
+  notify:
+    # This number needs to be changed whenever the number of runs in CI is changed.
+    # Another option is codecov-cli: https://github.com/codecov/codecov-cli#send-notifications
+    after_n_builds: 78
```

Comment:
You're right. It's
```suggestion
    after_n_builds: 27
```

Reply:
The higher number is coming from the scheduled runs that upload more reports on cron, if a commit stays on top of `main` for a while.

Reply:
Aha, thank you. I suppose there might be some way of filtering that out, though it probably doesn't matter much.
Weird that `78/27=2.89` isn't a whole number

Reply:
> Weird that 78/27=2.89 isn't a whole number

I also noticed this but haven't looked into why. It might be that some of those nightly runs fail to upload from some jobs (the current default is to silently ignore this in the action — we may be able to confirm once this PR is merged). Another possibility is that the scheduled runs have a different number of jobs.